### ISSUE TITLE
Silent login not working

### ISSUE THREAD

**Issue by ezazwar1** [EddyVerbruggen/cordova-plugin-googleplus#265]
Silent login does'nt work when internet is not connected on android phone 

Is there any workaround for this

**Comment by sammuggPS** [comment]
Check for internet connection and don't silently log in if you're not connected?
The plugin is not at fault for that issue. That's due to Google's SDKs.

**Comment by ezazwar1** [comment]
so what is the option offline:true for

**Comment by sammuggPS** [comment]
`offline: true` is for getting a serverAuthCode which is exchanged for an **access token** and **refresh token** that can be used on the **server side**.

The notion of "offline" access refers to a server continuing to have access to the google account even when the device has been disconnected from the internet.

**Comment by ezazwar1** [comment]
but currently the scenario is if internet is down and the user is not automatically logged in but if i click on a button that calls googleplus.login() function then even when internet is not connected the user is able to login properly but is their a way to check that if the user has already logged in with google and if he has then i will call googleplus.login() and get him logged in

**Comment by sammuggPS** [comment]
Sounds like you just answered your own question :)
If the internet is down, just call googleplus.login()! I don't foresee any problem with doing that.

**Comment by ezazwar1** [comment]
yeah i know how to handle it but as u know if it is users first time to login i dont want the prompt to appear on its own so thats y i was asking weather there is some way to know that user has already logged in with google,

One more thing i have the app on Playstore and i came to know that some users are able to login but their credentials are not returned properly . Like when the user logs in i send that info to my server but their are lots of users of the app that are able to login via google but their credentials are not send back to me , I dont know what is creating this fuss, Is it important for the user of google to be also a part of Google+ for successful login

**Comment by Barathwaja** [comment]
Silent Login not working.. getting **ERROR CODE 4**

**Comment by aakashchauhan71** [comment]
Silent Login not working.. getting ERROR CODE 4
